components/brave_wallet_ui/page/screens/composer_ui/from_asset/from_asset.tsx

```diff
@@ -0,0 +1,286 @@
+          {account.name}:
+        </FromText>
+        {isLoadingBalances ? (
+          <LoadingSkeleton
+            height={20}
+            width={60}
+          />
+        ) : (
+          <BalanceText
+            textSize='16px'
+            isBold={true}
+          >
+            {formatTokenBalanceWithSymbol(
+              tokenBalance,
+              token.decimals,
+              token.symbol,
+              6
+            )}
+          </BalanceText>
+        )}
+      </Row>
+    )
+  }, [token, tokenBalance, account, isLoadingBalances])
+
+  const fiatValue = React.useMemo(() => {
+    if (!token || token.isNft) {
+      return ''
+    }
+
+    return computeFiatAmount({
+      spotPriceRegistry,
+      value: new Amount(inputValue !== '' ? inputValue : '0')
+        .multiplyByDecimals(token.decimals)
+        .toHex(),
+      token: token
+    }).formatAsFiat(defaultFiatCurrency)
+  }, [spotPriceRegistry, token, inputValue, defaultFiatCurrency])
+
+  // render
+  return (
+    <Wrapper fullWidth={true}>
+      <Column
+        fullWidth={true}
+        justifyContent='space-between'
+        alignItems='center'
+        padding='32px 0px 58px 0px'
+      >
+        <Row
+          width='100%'
+          alignItems='center'
+          justifyContent='space-between'
+          padding='0px 16px'
+          marginBottom={10}
+          style={{ minHeight: 26 }}
+        >
+          {accountNameAndBalance}
+          {token && !token.isNft && account !== undefined && (
+            <Row width='unset'>
+              <PresetButton onClick={() => setPresetAmountValue(0.5)}>
+                {getLocale('braveWalletSendHalf')}
+              </PresetButton>
+              <HorizontalSpace space='8px' />
+              <PresetButton onClick={() => setPresetAmountValue(1)}>
+                {getLocale('braveWalletSendMax')}
+              </PresetButton>
+            </Row>
+          )}
+        </Row>
+        <Row
+          width='100%'
+          alignItems='center'
+          justifyContent='space-between'
+          padding='0px 16px 0px 6px'
+          marginBottom={10}
+          style={{ minHeight: 60 }}
+        >
+          <Row width='unset'>
+            <SelectButton
+              onClick={onClickSelectToken}
+              token={token}
+              selectedSendOption={token?.isNft ? '#nft' : '#token'}
+              placeholderText={getLocale('braveWalletChooseAsset')}
+            />
+          </Row>
+          {!token?.isNft && (
+            <AmountInput
+              placeholder='0.0'
+              type='number'
+              spellCheck={false}
+              onChange={onInputChange}
+              value={inputValue}
+              hasError={hasInputError}
+              autoFocus={true}
+            />
+          )}
+        </Row>
+        <Row
+          width='100%'
+          alignItems='center'
+          justifyContent='space-between'
+          padding='0px 16px'
+          style={{ minHeight: 22 }}
```

Comment:
Sorry I must have missed this before. Let's remove the inline `style` objects

Reply:
Fixed!